waspc/data/Cli/templates/new/src/server/tsconfig.json

```diff
@@ -22,6 +24,7 @@
         // definitions.
         "node_modules/@types/*"
       ]
-    }
+    },
+    "typeRoots": ["../../.wasp/out/server/node_modules/@types"]
```

Comment:
We need this for global stuff (such as `process`). The client might need it too, not sure. I'll add it to its `tsconfig` just in case.

Reply:
Should we add comment for this in the code also, since you felt the need to comment here?
Imagine in the future one of us looking at this line and trying to figure out its purpose -> is it going to be hard to figure it out? If so, let' add a comment.

Reply:
Sure, will do.